feature/subscriber-attributes/src/main/java/com/revenuecat/purchases/subscriberattributes/SubscriberAttributesBackend.kt

```diff
@@ -0,0 +1,40 @@
+package com.revenuecat.purchases.subscriberattributes
+
+import android.net.Uri
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.PurchasesErrorCode
+import com.revenuecat.purchases.common.Backend
+import com.revenuecat.purchases.common.SubscriberAttributeError
+
+class SubscriberAttributesBackend(
```

Comment:
from the name I kinda thought this would be a subclass of `Backend`. maybe something like poster or uploader might indicate the purpose a bit more? It might just be me, though